src/IL/OpCodeExtensions.cs

```diff
@@ -47,6 +47,11 @@ public static OpCode ReadOpCode(this Func<byte> reader)
             return op != 0xfe ? OneByteOpCodes[op] : TwoBytesOpCodes[reader()];
         }
 
+        public static OpCode ReadOpCode(this MethodBodyReader.ByteBuffer buffer) {
+            var op = buffer.ReadByte();
+            return op != 0xfe ? OneByteOpCodes[op] : TwoBytesOpCodes[buffer.ReadByte()];
```

Comment:
You are introducing a copy of a magic number, aka `0xfe`. Having an uncommented magic number is bad (my fault probably), however duplicating it is worse. This needs refactoring.